### PR TITLE
Reduce flake8 warnings 

### PR DESCRIPTION
**Pull request by rosswhitfield**
This should reduce the flake8 warnings by about 100

**To test:**
- Code review

_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
- [x] Is the code of an acceptable quality?
- [x] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)? Is it well structured with small focussed classes/methods/functions?
- [ ] Are there unit/system tests in place? Are the unit tests small and test the a class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?
##### Functional Tests
- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] How do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant documentation been added/updated?
- [ ] Is user-facing documentation written in a user-friendly manner?
- [ ] Has developer documentation been updated if required?
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: scripts/Interface/ui/reflectometer/refl_gui.py [hunk 1]
```diff
@@ -743,12 +751,14 @@ def _process(self):
                                 logger.notice("Calculated resolution: " + str(dqq))
                             except:
                                 self.statusMain.clearMessage()
-                                logger.error("Failed to calculate dq/q because we could not find theta in the workspace's sample log. Try entering theta or dq/q manually.")
+                                logger.error("Failed to calculate dq/q because we could not find theta in the workspace's sample log. "
+                                             "Try entering theta or dq/q manually.")
```
Comment (KarlPalmen) [inline scripts/Interface/ui/reflectometer/refl_gui.py]: line still too long.

File: scripts/Inelastic/Direct/ReductionHelpers.py [hunk 2]
```diff
@@ -228,10 +228,12 @@ def build_subst_dictionary(synonims_list=None) :
         if len(keys) < 2 :
             raise AttributeError("The pairs in the synonyms fields have to have form key1=key2=key3 with at least two values present")
         if len(keys[0]) == 0:
-            raise AttributeError("The pairs in the synonyms fields have to have form key1=key2=key3 with at least two values present, but the first key is empty")
+            raise AttributeError("The pairs in the synonyms fields have to have form key1=key2=key3 with at least two values present, "
```
Comment (rosswhitfield) [inline scripts/Inelastic/Direct/ReductionHelpers.py]: The `max-line-length` set in `flake8` and what was set in `pylint` is `140`. What should it be set to make it look good in github? and do we care?

File: scripts/Interface/ui/reflectometer/refl_gui.py (hunk 1 above)
Comment (KarlPalmen) [inline scripts/Interface/ui/reflectometer/refl_gui.py]: There are others too.

File: scripts/Inelastic/Direct/ReductionHelpers.py (hunk 2 above)
Comment (KarlPalmen) [inline scripts/Inelastic/Direct/ReductionHelpers.py]: I counted and found that when I view the whole file in GIT The first 140 characters are shown from a line, but when displaying changes only 133 characters are shown.

Comment (KarlPalmen) [inline scripts/Inelastic/Direct/ReductionHelpers.py]: I think 7 characters are deducted for the second column of lime numbers when showing changes.

Comment (rosswhitfield) [inline scripts/Inelastic/Direct/ReductionHelpers.py]: The split view shows the entire line when showing the diff

Comment (KarlPalmen) [inline scripts/Inelastic/Direct/ReductionHelpers.py]: Ah! Split view wraps the line (indicated by absent line number).

**Comment by KarlPalmen** [comment]
:shipit:

**Pull request merged**